src/node/interfaces.cpp

```diff
@@ -514,6 +514,11 @@ class ChainImpl : public Chain
         LOCK(::cs_main);
         return Assert(chainman().ActiveChain()[height])->GetBlockHash();
     }
+    int64_t getBlockTime(int height) override
+    {
+        LOCK(::cs_main);
+        return Assert(chainman().ActiveChain()[height])->GetBlockTime();
```

Comment:
fec6a0e9c7c4d98a88628ab57ce33144923da85d

In `Chain` interface, probably better to use a type-safe `CBlockIndex::Time()` https://github.com/bitcoin-core/gui-qml/blob/fd4e57f63654e2d8e7d0de0aafc335715a53a5cc/src/chain.h#L279-L282

Anyway, it will take a broader discussion when it will about to merge into the main repo.

Reply:
Sure, let me look through it.

> it will take a broader discussion when it will about to merge into the main repo.

Let me include that, along with other future suggestions for this file.